app/views/admin/invitations/_menu.html.haml

```diff
@@ -0,0 +1,21 @@
+.ui.fluid.vertical.menu.sticky{ data: { behavior: "sticky", context: "#stickContext", offset: 75 } }
+  .item
+    .header
+      Invitations
+
+  = active_link_to admin_invitations_path, active: :exclusive, class: "item" do
+    Index
+    - if count
+      .ui.label= count
+
+  = active_link_to new_admin_invitation_path, class: "item" do
+    %i.plus.icon
+    New
+
+  -#.item
```

Comment:
Aaaaaaaaaaaaahhhhhhhhhhhhhhh the code! The CODE!! IT'S...it's...COMMENTED!!!!!!!!!!!!!!!!